setup.py

```diff
@@ -72,7 +73,9 @@ def findsome(subdir, extensions):
         'patool>=1.7',
         'six>=1.8.0',
         'wrapt',
-    ] + pbar_requires,
+    ] +
+    pbar_requires +
+    (['colorama'] if os.name == 'nt' else []),
```

Comment:
I believe you could also do 'colorama;platform_system=="Windows"', which I wasn't aware of until recently.  Not sure if there is any reason to prefer one or the other.

Reply:
We use
```
platform_system = platform.system().lower()
on_windows = platform_system == 'windows'
```
so into better to be consistent either way